From ab149a0feddf32cd657c9d210e15839a451360bb Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Tue, 27 Oct 2026 12:09:07 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the direct GnuPG verification with %openpgpverify, which
uses Sequoia sqv, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh kresd-keyblock.asc from keyservers/WKD: same certificate(s),
with self-signatures our copy was missing; the commands are in the spec
comment.
Release bumped with a changelog entry.

Assisted-by: Claude Opus 5.5
---
 knot-resolver.spec |  15 +-
 kresd-keyblock.asc | 447 +++++++++++++++++++++++++++++++++------------
 2 files changed, 344 insertions(+), 118 deletions(-)

diff --git a/knot-resolver.spec b/knot-resolver.spec
index 9b49ee5..fd093ca 100644
--- a/knot-resolver.spec
+++ b/knot-resolver.spec
@@ -7,7 +7,7 @@
 
 Name:           knot-resolver
 Version:        5.7.9
-Release:        1%{?dist}
+Release:        2%{?dist}
 Summary:        Caching full DNS Resolver
 
 License:        GPL-3.0-or-later
@@ -23,6 +23,10 @@ Source1:        https://secure.nic.cz/files/%{name}/%{name}-%{version}.tar.xz.as
 # Export with --armor using command from https://fedoraproject.org/wiki/PackagingDrafts:GPGSignatures
 # Don't forget to update %%prep section when adding/removing keys
 # This key is from: https://secure.nic.cz/files/knot-resolver/kresd-keyblock.asc
+# Refreshed from keyservers/WKD (same keys, newer signatures):
+#   sq --home=none network search --output - 3057EE9A448F362D74205A779AB120DA0A76F6DE B6006460B60A80E782062449E747DF1F9575A3AA | sq keyring filter --experimental --cert 3057EE9A448F362D74205A779AB120DA0A76F6DE --cert B6006460B60A80E782062449E747DF1F9575A3AA --output update.pgp
+#   sq keyring merge --output kresd-keyblock.asc.new kresd-keyblock.asc update.pgp
+#   mv kresd-keyblock.asc.new kresd-keyblock.asc
 Source100:      kresd-keyblock.asc
 BuildRequires:  gnupg2
 %endif
@@ -42,6 +46,7 @@ BuildRequires:  pkgconfig(libuv)
 BuildRequires:  pkgconfig(luajit) >= 2.0
 BuildRequires:  pkgconfig(jemalloc)
 BuildRequires:  systemd-rpm-macros
+BuildRequires:  openpgpverify
 
 Requires:       systemd
 Requires(post): systemd
@@ -152,10 +157,7 @@ native C implementation, which doesn't require this package.
 
 %prep
 %if 0%{GPG_CHECK}
-export GNUPGHOME=./gpg-keyring
-mkdir -m 700 ${GNUPGHOME}
-gpg2 --import %{SOURCE100}
-gpg2 --verify %{SOURCE1} %{SOURCE0}
+%{openpgpverify} --keyring='%{SOURCE100}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %endif
 %setup -q -n %{name}-%{version}
 
@@ -374,6 +376,9 @@ fi
 %endif
 
 %changelog
+* Mon Oct 26 2026 Jakub Jelen <jjelen@redhat.com> - 5.7.9-2
+- Verify upstream source signatures with openpgpverify
+
 * Thu Oct 22 2026 Jan Včelák <jvcelak@fedoraproject.org> - 5.7.9-1
 - New upstream version 5.7.9 (security fixes)
 
diff --git a/kresd-keyblock.asc b/kresd-keyblock.asc
index 1b34475..e2a66a5 100644
--- a/kresd-keyblock.asc
+++ b/kresd-keyblock.asc
@@ -1,6 +1,6 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
 
-mQINBGImJYQBEACoQzmvkGLYGeltnY2MzvjZlpZk/M670+vxI6e7ofto4AQQmvGW
+xsFNBGImJYQBEACoQzmvkGLYGeltnY2MzvjZlpZk/M670+vxI6e7ofto4AQQmvGW
 0eann8bUl8qZnliOvqq57akCRTYLaT9ArFjfaaYs60qFD7KCFTAe4GHry/J4aWiP
 Q7jftfpDerTLr+LvikJ9dJrMLV/TA79fgEb3DcGOtuZMventwJbZKS5iiAgaN/mW
 ZZIh/kQNvylKy4aTm21yx2XFx4eVDRd1aucJWTrR9DmM3Nm02rpNvQxjxWfDU0cg
@@ -11,116 +11,337 @@ yP9MwKVNzEkgtxeWvjSmKCpqmmD8oiXO1iL5bcZsJfJH1p9nHqDVG4+EJjSbu+wr
 eM424BxXap+bdvoxKr5ofTswEyvlwn47aejQQVciDDb+AN8nufJeOrfyYQ798DXj
 PhkR3CFVGfLCGkAAoTrhEePmn1DqKJMxlDB8YSZKTvF+TNBLVpVExu/MsVMX7mMh
 uG+OFrDn/mCFe+q0fqMEQOpa5XFzhzaRd93khifR+sYnF5fiE92nG60psQARAQAB
-tCBBbGVzIE1yYXplayA8YWxlcy5tcmF6ZWtAbmljLmN6PokCVAQTAQgAPhYhBDBX
-7ppEjzYtdCBad5qxINoKdvbeBQJiJiWEAhsDBQkDwmcABQsJCAcCBhUKCQgLAgQW
-AgMBAh4BAheAAAoJEJqxINoKdvbe5tsQAI4KnAGf+YkPGLdCTQo7bi9eww3zVoJk
-m0WKTf67zLd7JC8puCGPWRVaVSBsuUKDrp57I3h736+Jx0XiwRA6xlFmgvVIWrFr
-swGXlLbdutezXE2ke5y+86uj6k5TQKn3jEWfmkRigFLlUy5U+VJEt6b/i7e9EmMb
-T9dI4d7JW7MPY74jYlFJ4klZvp/BL5ldz9R2yprOnwBX4Bjwm4FFR71NPC/iK2Dn
-19pSH+YeRpsTU1I1vBMzz4qlEkhhE1+pqYbxYpyfkmSxJxGvD7FFr8GHnFwcnqqv
-zZuinIz+eGf7iSql6yxz7OxDnVhBYnSQpCkSWtGdhHEVtQo0uA7gcxcdz5URBScQ
-ofR/Zs/fvEvWqRXKm+I/qC1nK/glfaoA81xI6OGOPkrYEoLi9WKGK0nWSbnux9qY
-/vyzVAKUDjEtYFm/Nk/23D4tBcC0/tZfuVFedWa6rcqKVNoeRdu2Nol9L4quL601
-vm/kIQDhUsQmWbvIkwx6pHZoFw+9COY1NFINOqguBYzyMIlVJpo6Xns+1gldJ7vI
-0w6F+wpmUkFNnEACQ3dG/uf6WNzEbgEdKFQ2xgLN4zY7nJI3QLRlrXntk6BzG7Lj
-iQe7ZS4uhaQQ9bzTuvhi3qKJEJUGaPQRvYJoCoDQs1gLR/rep0qQpRsUt7Tzb0uZ
-xeeZHSvvhDq5uQINBGImJYQBEADaE8cZEkr06+TnlQ+8qSBHRFJSEunNKjUr8t0y
-zwPgFZhRIRtw3mCFqDEv/eBPw9tsQeW6aJz5+VI5t7Ifkt8krHPE8GT5N33MyUKj
-8sRk1oQNt/2JdDCOy5ZigyqFj31/9XeIWvlQbBLUTL0V3EPAmcipegEx9VFD81FW
-qH4TbBer0oLsa7zFnZ455FIn1a76UNC+73DE3+eV1V51euhx3TIgn76SXe8Qf+jg
-Asv7Ylfe++g3Smkt5UAgEZQOFwPbYMjS4U5sRq8hpJw+nBkI2n/vHoLeSKWZjt6J
-/yth+tLQmsQRwyYk97Tp8mzPpmWSMWIGPDrfyBdV1DmToQkWDozuBs5Mz8gVvrMu
-2mNe4O0m5Xv7e80Gm/OJxSeB/o8tEXFE12JEM1c88OZ8Iz/PcfSgotbcS5zVxRcm
-yU/ZqMRNvJHtG932DIhZ/s0NdAxnsAeCz9gaR6Z4orVXb8G7/S1pe0ab66k19VX5
-wFoPIz92SUq91VbrCsjfHi4FeE7h2OL9QP10Liv9qOxJFYvy2fvWG9EXuknjaZsD
-igmC1DuX/7uWJbrGbp9spnuNsgm7wYG5nFH8fTnNKtaMl2/D8D5WbB0k+MJnBDeW
-MyDKPsaCStX2ih90DGSOkWlZy1k3X+v+PTPPxu1ry/MePSo0UHud9L2oWLbnz27I
-bBRymwARAQABiQI8BBgBCAAmFiEEMFfumkSPNi10IFp3mrEg2gp29t4FAmImJYQC
-GwwFCQPCZwAACgkQmrEg2gp29t7Trw//cN9UEaN1WaH4jFDKIST+xdLnSLvk0Izt
-hHqJzzW9sRQYINvqz89EkTUrjelbW3Ib41YVNNk3S7hOm5OJm/fRHdQtItzuY/9E
-RafF5+PUiDj5TRDa8bRV7sCAQLoztfr1ozadbGE7e8HrrZv4DcZ10joCpLRZm04j
-QMBeZcaaEa2Wh7Xf9VW4aBJ/DU5DevsKeTRpCuwtw2lEf+NMl3PogPa9glm7liQu
-DKU4Zlf/31ZCudAb80PugOzWM2i1DG26G/b7z+8N59m6dBYWeSvcqU7Z9Q4rU7bV
-R/6nST8GyLIU/PsDalvKm2B3NDcXerN8lT9zU1QBzT3OyL6Uy/UzXl6YnpxFT8m/
-Gm4uiLS+gWD2TXh5BKGsuU4zIHakjglA7Gtb6PJ56+Pkz95apx7vAhmnJxgw7qAX
-KAH0Rng7IxEeL8Fj67tzWTQPs0PuAty7qIgbeft8hvhtk9vfFZb8k3bhX59b/+TO
-tbTOCjVhmq9bZdaw60SBl3LuyUlnlGmrqvPzVUdcAc2zvnxjEt9yVioS/M8zNRC/
-SmWbagZ/xyf2iiEAbo0y12t3heztdG8jMoMrImmUEubfPiAfCN2ys5zPiXW2jio5
-vvopiu23y37ew3zoAZrnfmFW0D8A8iD2g0kl/Ysb2dXIYocQ423ADN1Em/+1f8j5
-3P6DgfI04ROZAg0EWAOSdgEQAMcRDAsGV+ptULOruphwEUVNzVYIAW2qIp6E6uwh
-Ocq6xnDieFjCUpDFy6aCPTXEk6Ft8cEIF4aUPJ1ip6YqDgh9jafBLI/Bllgwmfsv
-sAasJ7K2vhUy8MnujuoNmOOKbiJ+/4ZTB2TemVfHqgwXidy16wjxoA64tQcCfQ9+
-sZ82FahqCU4qB04joMaOEj9oaS/L6Osj/5ex0xX7d/VG4Q58j5i7hJcWMnKECaoQ
-4FHHF0eoqt4ginPcuDfzJbCJUZ/CBBP6cUjP0GkcfeVaDW5LzEzd7Nx4GZFhXzNP
-0+9t5nnsGVpNHqstp7lFjBXf3s9IsGqouQHkfRMNFQUGfAGxShst5tgeYkI7J2/I
-2/kAWQuYtns7iPaMWjVY9Iy3dOEzrBHCqucHgaL7qifKtl8TUOdVkypvFndVdr4T
-hLljCt8ItQSsYSf53jQ62hFmZBeFE9UCx2OKYyP2gUz1GjFKBSsODohrefQisoVv
-77McAVuqRHmDirNeI4wQbrbUZgR7HawMQrP88PgJUqU0j16V3hOgfn3aQDDMmZ36
-wCL5F+izQ0TPG4V3R6QA55ahRGjTOhhNJY+Uw6YUy9ykfos/eV4mZjvHjP9vWeD/
-7tScqidUAtU8VtFsSU6Qh8C0spxQSh/BXWOvsPJgY9qUm2IMk+iYHTjVkcKPZHHJ
-dj+hABEBAAG0KVZsYWRpbcOtciDEjHVuw6F0IChwZXJzb25hbCkgPHZAY3VuYXQu
-Y3o+iQJUBBMBCAA+FiEEtgBkYLYKgOeCBiRJ50ffH5V1o6oFAmFmsBgCGyMFCRTR
-36EFCwkIBwIGFQoJCAsCBBYCAwECHgECF4AACgkQ50ffH5V1o6r5GBAAxkNaB8zi
-TVWBZ5geqcQeoqCeTrZjDZ1pY32BbjefGb0z0fmRzH9ESQBGH+b46PO7cibOt3sY
-d4ZX8GxG/vnwsU3dOspaIQfHH99yXr+sCRkcr2k7DWTMFSX/5JkhlrouAURpIRN8
-y4+OO/TkWFY3HAIEy3VkhwIoJSUEDfSINb1KHnotNzHc7s1Z3yb14K14XV/LnQL/
-/NTz32NBGSDAt3C3+w4h5WKhg+kPcuno1VVUHM4AcqqLkhEbNN/QwQULWbrQMqcB
-+JJWlEgSqkhn2nrw/i8q+tlK7FBi4DhID99xRljt2HcPt7gqhnZ/2OxfoHGHui/f
-FPgmeK+c/wHZ22FH4ly0XwsSdIjskhRf/XPTaBlgeorfyIEBFBeFkgjdFUNSpl1M
-Tso6ZYxXkLg1UxEFtI+X5Eh7zpirfs6jwc44aMHKabJY3V/MBmYqUBRR3H6xnIFd
-gfjVeFAR65w7MElJXRx2kvJ5ixpV5W2+U//MvhkpN6Vk1Y2G9NXmQeGXz2l+2A9V
-4qpTn/nVAIchYFtVPzsAJlyPAqB7I45uNMQOcsGdn4W5lLgD4MNdgwIUce9jQ1Oo
-F0BnZSCJGLjU0ZxVUOd7fwF+P+Y8RBxBIhVcvxaluI3YmT0roqCXaJaYGAWgXZCF
-9g0NnfvR0dRRZRUeT/qLxJNZksK+iFtLIlu0MFZsYWRpbcOtciDEjHVuw6F0ICh3
-b3JrKSA8dmxhZGltaXIuY3VuYXRAbmljLmN6PokCVAQTAQgAPgIbIwUJFNHfoQUL
-CQgHAgYVCAkKCwIEFgIDAQIeAQIXgBYhBLYAZGC2CoDnggYkSedH3x+VdaOqBQJh
-ZrAYAAoJEOdH3x+VdaOqVjIP/1N5v2soWf7T2OcromFmL76QDqAIwzYtuwookFK8
-UBiGEAw+lz8dHSPqiQF8IiaVxroWeKp7vHKjSeRCr2a9eKsYzTdz3jStwPnKZGpP
-5OMJb2x5dZNxyKaL1Fyidq0N9LDIkxanMORPYVA2c1tHmFJFJoj8EmUhIYN8iAl5
-ZG2u3vELI9B3ZvNJb8OJUxAOcvPVyuxKMzpJY2C8K4Ec2joLugG+aCoZh7DTi4EO
-tqAOxr/jfCPEdPHEIKEOdqXy5DtoiA5e6uHA+DpseBC/jM2jrvO+IRqnFvr7M7+Q
-yfv0ZbkFefsJp5tjojCv6xJPtz/BKwh9wxNtUJrZfFL71Otz4/0YAwiX4z+kljop
-x0i4ALVUOiWb7i/2PiP23n4JnPAfLdlClk9CeTkCOu0CAgBvrKi1w/7eEnGqOgTu
-4OaGfFaOtTjuFqKS0wOdBRGST5h3pKjxrDxN8HgI+ZpF1EfMd6ovfLWpMVIwE9cV
-ueimcpAqfSubQECTjwnsDSmzc26apaCHL8FaXf2pG9J5uDczxdcUzVWfgb+16Wdg
-bFoqJ82O2SCysTcEkrNH8m2R5i7iCaO5WlQFLf2FHOBWLd3RjV1ztCImv4di5skw
-QLwgW2+wi0QowVPgPc38JAPa73INRhaPK3JUcwqNEr9JUNfRGyL+eTwTX4b5YKHd
-dJ9KtCRWbGFkaW3DrXIgxIx1bsOhdCA8dmN1bmF0QGdtYWlsLmNvbT6JAlcEEwEI
-AEECGyMFCRTR36EFCwkIBwIGFQgJCgsCBBYCAwECHgECF4AWIQS2AGRgtgqA54IG
-JEnnR98flXWjqgUCYWawGAIZAQAKCRDnR98flXWjqjpMEACClbUGJ3e0p1Nty0Lf
-kRsoWZMoyeWSoBA0owUsQOyquKKaSj+40qHMDOhbNfJoLpwFsU5zJL4I2rhDlW+z
-Y5Het5x1afS0lExzXK3RMPWwhUwMmWevUpW+VFjvPyBhdlEP8E0Ayj6z/p5GEN9Z
-9yruZfz9ru48KDvwUYz/f2V0cAPqOIcSM6euGLXnK4ExML+8Wiqgmht8qBozD0Lo
-odyAWMR1D+XkAMR+ugwUfs449+x7zzzfkRh8CfggB9mgMK0i2b66MdWFHYW2c9jc
-3SIEaWZM56pEG9BCZpcoiiGgCOMWQ7CEn2dOfH2Ah1ZiICyDnL9XvESYJBoRLiNG
-cLrD31GJzUrDPRLzK9f8j4AlWN7bNECw5pmPYE2vizFOuasojBV6hlg+c0bXpQSK
-ybg1eQb0rE5yOWrA0pP5yNc6ZIBxtjbYLbwmWltb9mWMznVSNfcjvA7QXe3srYYL
-AOW1KPz2Ly8qCTZXcpXx2Z2mk4FTNDIj4maQxLTIURHWHckHrXxENxMXNcbaZ4d4
-LVz5f1RBhsFfmAXpVu/FTvXIPlIWaNVlQDWBMxlVidpeztB4XgPUqKWmtMNyCTsN
-PeuxrRiyIm2ccXE+fcuPDstLIeQdX+U/rijx1f588i6nt2oD0vy9tEchFeiczql1
-YzYNM5U8u/rCYiOxmQWf7gZecbkCDQRYA5J2ARAAyHww3huLEtsdyqgjiGMhtEKO
-Lmp7yFl450HY9oPcHS02U5BC1370ssNShrdOCi2ACDbe41Zxx85WcuaO1OVqung2
-umX047mj2xQsiTAFRDLZsQu8cQFoEy/DBL2bk7ThfK1Lh+NyZAs0UaPpDkGodS0D
-e9osA+4T6Nf4POYaeavbYVFSdDKS4lUboBqApKnD/TzKFxFcpuFx6FN92lteTbOo
-jGMiLoZvELY86Kn9KuFZ8FM2ZSNHx1Z75KouufGrdkeCoZYVYiuzT+fnt2it4dIp
-IlnF+yxMt5LB/MSrmECB5CAFJtxzuMccm6yDUZQSWWi9vUgxIJwvt5w0CIBT353D
-GeP4WnH0r5YoBKoRbh7i4fT0lWvMXTG/V2lqyzBdClMebyHffMgba26Kj6oeDygD
-fC5aGsVaqw1Ue/qQ5QRqTJcJV7xVLTtS1EamVqkfKwPS0zTfnrF1jQtnO/P4qkfg
-BRRG9BXGGrykHpXOyqmX6Z0wbV2P4j+p02oSecDl5yVXplJfsXfbS/xXnaSkaN/7
-mCU29ul26cAVNxDkDPunztSFi9K9LM2T/XWYJQGXM71OpmONQJGF24lx7Wp/kobn
-HtbjGDzjDPC4eSL7MA56qtrWaLM+4ePKANct2q0q6c0uSLs0Q2zochS64Mcg0YzL
-1sinWPN1rXLDk3lwpIsAEQEAAYkCWgQYAQgADwUCYWawGAIbDAUJFNHfoQA/CRDn
-R98flXWjqgkQ50ffH5V1o6oJEOdH3x+VdaOqCRDnR98flXWjqhYhBLYAZGC2CoDn
-ggYkSedH3x+VdaOqtBgQALeChWgHEGe/8nwKWrC8CMQyKyiJRlSfvERi5M40PYxw
-KC7IHo+ekMdlLc8kVkv5WSq3zQcNoRMjHutZlCVpGJat5PwiCine6I3Z5JfKsU7f
-JE4KXPD0jr9kCy/IHlCWsKLZTHH0LOk98tqZcFQwFM5nQEzu8Us86BxlZs5IaN0j
-ILD2qUC+EYeg0hd0kya+16UNao9NxCeUEoSwxhtB4IIMCJTUPx0pWErCHxeCUjvx
-Pd2h5A106DT/3fa8uhPHr9goHtfreV0soGuxu1rqGrMn7XcorsGdt9XX4tyPHcXD
-XCZMLedjYTu7OcYQ63hi+ZWUXJPvcIrB4StCumaFECwLodpdDWB7n6OF8r+X//eP
-bjS8qt++A20LGD4l7BrWxDeymrnqLmWN8RBf2xV4ytVfcRx1ercFMGiNRUvGSh+k
-gP6J7D4yT4xGTq+fCtS6BHyWMB0loXcWv0Im/6znpIpUcvXyL68s4Jiqukt94rKl
-wb/IB6MSWannxk8UzWfGgFXeGUrR8YWLuQSo26AceNOyu2gMW+k9kYi0sqkBzMPk
-1zWZN/gSNpDyu7AWGpv4RtOKhJhaux8zwtwqozqSx+plpnYz5ifwCNMuznM6T0j4
-msKQiK1N7lIwKBGaT2P8SJ/ecaJyudIz4ds2+GUl6fxqSR27Egs+EUY0BmS7wESr
-=dRuR
+zSBBbGVzIE1yYXplayA8YWxlcy5tcmF6ZWtAbmljLmN6PsLBlAQTAQgAPhYhBDBX
+7ppEjzYtdCBad5qxINoKdvbeBQJpg159AhsDBQkO4nfsBQsJCAcCBhUKCQgLAgQW
+AgMBAh4BAheAAAoJEJqxINoKdvbeyrkP/i5K0NPB2QuFXblqQJJKtYa7zaNyoPDU
+PHmHaA3eQPledzTULRGhuz2Ru9HHOd7eavUITtslhkM/0JGwwNsxILKFPmiYbTDS
+wSoNgdPLjhh977/oBvtf17kyNvSU1gwIw5S1orPMWLm+AaBjGc3UXJ+fPW9NBePe
+H/IeU0/NUPYCkKkbS8IWtJLIYizyAwwNaAx89TcdPgn2xFKVOsawdYcBTWCGcis9
+H3RPWlRY4i5wKNL2Y7TJ2WLDGzIj9jH7SXXWrevmE7nMq2NRttvUAiKQRvDPdva7
+V6FQcYgY5eFdg9wxvN5EW/xgfwRhvYnR725ojr4KWaH+TCLe9AC6fql/HFGw1jBy
+slH/3fDF/zm8Ew4H1B6H4zBKRNP35QFocyfgvqx//mt7NOVLeLMUNkpnjARBXT1j
+IqjO85ktn2wCREViXDhVDWO86sH7863wFJx0MK0sBADKkaMkzJiQQhvTVWFZ6aXZ
+HCEQ7Qb8dZ3KKBfjqBBkCGzjBJndEO1NGmkGZ0i0kIb+qrXSp5yzlB9JoE6A2FEG
+YkcYudxpVS5jF0TI7zBOIAxMJ3ZP4wif2pQyTV4qZLYzN9xrHYNybiZiKThY6XMP
+U6PAS/8tFleFfARGcZKWRrYn6CFAt7hmnnsURyo3XJoFoy+X9dM4TsQhxJqCx9Cn
+JZVVXM6xOexEwsGUBBMBCAA+FiEEMFfumkSPNi10IFp3mrEg2gp29t4FAmXllIcC
+GwMFCQeBQmwFCwkIBwIGFQoJCAsCBBYCAwECHgECF4AACgkQmrEg2gp29t7s0xAA
+lc/bQ6+8iTkwwStPkiItlNNtceyIcFzcxnpr/sQF9ZlQ30Jv5FriQIh2A1t0sLCR
+CcFheKuRfjaQnKyYpcxzaIhpf9BKmJO+sSHChGkv7n8pFSWAbr/hnOFZBL4ysN/2
+L5WfGnx2uhp+u+VGlgRsXuWdvi3tOZwI2dcPRdw0uosMLz8+s3yIKdAhI/Uh3yhN
+4qzJSF3h2hJYYVGeuiv+EtUTZvhg6nV77qUwtiRe3+HlMlMLVpwDfONds0sZNi/I
+ZOVe4TZV2idFAQlE1QMqX7ShUKj0PBseOgX17ZsFNqJya2QCPyhsmqPFyi3z92St
+g/BRyAskT4urNnzppoLxcPMBkrRSKSBE+omO3LvztZFfbaVI38a6PR7ROWt0PwgH
+cT9Be+oDDAh+nKTTKtlgHbPB4kJo0u+MqZE4v7NdHvREkkJ9k4qGX5WggGTMQSiV
+kdQCrhQmMg7FE+cdQola66qox7pT6a/ScPzhYWVQYFZRVB6CgSQcWhWu/PJpqxmc
+FkGKZUEgNtfDkRXEpID+LljcSsTN7AccPJVEfZV9XDKeZuh/brUcUYjCzo7pG0QL
+CKJQCB7WovMP6Wl29ftrwVa7+7pAkLYFOhP6KQWkE68XJHVtiigDXEyZbwJfaeb5
+ltVaGMUWp0Orcb+4ytcyyhQXFkv1jZ9pEQzoS1vYWQ/CwZQEEwEIAD4WIQQwV+6a
+RI82LXQgWneasSDaCnb23gUCYiYlhAIbAwUJA8JnAAULCQgHAgYVCgkICwIEFgID
+AQIeAQIXgAAKCRCasSDaCnb23ubbEACOCpwBn/mJDxi3Qk0KO24vXsMN81aCZJtF
+ik3+u8y3eyQvKbghj1kVWlUgbLlCg66eeyN4e9+vicdF4sEQOsZRZoL1SFqxa7MB
+l5S23brXs1xNpHucvvOro+pOU0Cp94xFn5pEYoBS5VMuVPlSRLem/4u3vRJjG0/X
+SOHeyVuzD2O+I2JRSeJJWb6fwS+ZXc/Udsqazp8AV+AY8JuBRUe9TTwv4itg59fa
+Uh/mHkabE1NSNbwTM8+KpRJIYRNfqamG8WKcn5JksScRrw+xRa/Bh5xcHJ6qr82b
+opyM/nhn+4kqpessc+zsQ51YQWJ0kKQpElrRnYRxFbUKNLgO4HMXHc+VEQUnEKH0
+f2bP37xL1qkVypviP6gtZyv4JX2qAPNcSOjhjj5K2BKC4vVihitJ1km57sfamP78
+s1QClA4xLWBZvzZP9tw+LQXAtP7WX7lRXnVmuq3KilTaHkXbtjaJfS+Kri+tNb5v
+5CEA4VLEJlm7yJMMeqR2aBcPvQjmNTRSDTqoLgWM8jCJVSaaOl57PtYJXSe7yNMO
+hfsKZlJBTZxAAkN3Rv7n+ljcxG4BHShUNsYCzeM2O5ySN0C0Za157ZOgcxuy44kH
+u2UuLoWkEPW807r4Yt6iiRCVBmj0Eb2CaAqA0LNYC0f63qdKkKUbFLe0829LmcXn
+mR0r74Q6uc7BTQRiJiWEARAA2hPHGRJK9Ovk55UPvKkgR0RSUhLpzSo1K/LdMs8D
+4BWYUSEbcN5ghagxL/3gT8PbbEHlumic+flSObeyH5LfJKxzxPBk+Td9zMlCo/LE
+ZNaEDbf9iXQwjsuWYoMqhY99f/V3iFr5UGwS1Ey9FdxDwJnIqXoBMfVRQ/NRVqh+
+E2wXq9KC7Gu8xZ2eOeRSJ9Wu+lDQvu9wxN/nldVedXrocd0yIJ++kl3vEH/o4ALL
++2JX3vvoN0ppLeVAIBGUDhcD22DI0uFObEavIaScPpwZCNp/7x6C3kilmY7eif8r
+YfrS0JrEEcMmJPe06fJsz6ZlkjFiBjw638gXVdQ5k6EJFg6M7gbOTM/IFb6zLtpj
+XuDtJuV7+3vNBpvzicUngf6PLRFxRNdiRDNXPPDmfCM/z3H0oKLW3Euc1cUXJslP
+2ajETbyR7Rvd9gyIWf7NDXQMZ7AHgs/YGkemeKK1V2/Bu/0taXtGm+upNfVV+cBa
+DyM/dklKvdVW6wrI3x4uBXhO4dji/UD9dC4r/ajsSRWL8tn71hvRF7pJ42mbA4oJ
+gtQ7l/+7liW6xm6fbKZ7jbIJu8GBuZxR/H05zSrWjJdvw/A+VmwdJPjCZwQ3ljMg
+yj7GgkrV9oofdAxkjpFpWctZN1/r/j0zz8bta8vzHj0qNFB7nfS9qFi2589uyGwU
+cpsAEQEAAcLBfAQYAQgAJhYhBDBX7ppEjzYtdCBad5qxINoKdvbeBQJpg159AhsM
+BQkO4nfsAAoJEJqxINoKdvbebRkP/0082v35r7bF/U474cJFMqnp/22Xaykzmsyq
+bYQ3tilcVu2wa30kAbdfEaBnVAyB2kZXj6TvrjaF4aylPbJuZE42bOkOndgzQsgF
+MBBBuh68o3svDKlyj2NgmPcrefGh5IpPBJIIHeNkGP0kFkuGUx4lQapHEVsIQ0Ec
+ekP6xSOA/MPvJoC15Sm0PziAHZrMRzjxBXeOy0/NJVFAUzqD5MGd7TDqXbgAKWIg
+BaAXCphXxGGKibUDBjb7j5jRMVqgJOdRvn3Xk6efV5SHxGCHD/WH0FsOYO056Cab
+45woLP+gwdFT/1fx77JEDHIzX6BzoPjHz7ZyKYT6wyjvx6cYE+Hdq1b1QOietDfv
+c3mYgxQoqjKk6+d+cBRV7NZpgeaggqQg3O4FlxTGBIuKzf2Nsu2wWeWwBs91aOnq
+gyk4Qxw1idXwgHx/tMaXu8bo2pWjqrgSKPwoVSfox0Lg0b3CBADfLdf4g/FlyZbF
+H7ur9Sz43EfXjIg0N8zHPwfKGkeD55f6OytmRlYptkJV4E2pSqh8fsTkmDRl4Y2N
+iglVkNE1hA1yX1O/QuCiykdyYD+dLZmxb6Fhe7EJPW09QlF1bjUircka74G/J39s
+CFdcCd54NquA33912zZP663GLnQ4KHcG115WD8W5aKRIsybUUBemzte/G8dsYhjC
+kTOJM9P2wsF8BBgBCAAmFiEEMFfumkSPNi10IFp3mrEg2gp29t4FAmXllIcCGwwF
+CQeBQmwACgkQmrEg2gp29t7dkg//TIKucPoqGNikDyA1sNdx/ErdGH6GaXIS47AI
+8BNS0cvUqFRUufe5nxFZ7F4+JRUWyFMrg5OBLM0Q0+4iFHkt2tmRMi5GTSfE1BHo
+OaUfj+YKz+qN6w3/vqhnii7TaYy9CJdjmfmoXE2ow3UBUUmy+12OgH4lpyZfseGN
+iUZ+xTmrDj+N7D5BrgISyVRyPvfY45l5m8Fa+i3iD5Ek90KMQGxXMc1Rcyeej0n6
+aV6dZ1Y72jRTer+Z3tYIWqDPjtxZAnh9J7TeIRVCBOkFaliai/cHKyOWV/WSV/1l
+Sk7EeSfTpl+PMxzA72H5S1mYKGcwPkz2GziBLfExEjeKjh3zU2JHdquzRh2WleKk
+rn4YkV4H9uoKwfCwkupnoCinUQA07Ejye+WB3k63Pb/YRMCTDXw9yji0WsoYN120
+Y6NgIZvM9n0f4Gm6Af2RDgf2P1CatFaZf+yXV2EQnAIWBJgRe2xiPt/XAa3YHu0n
+8aS3vzVoXk62d8WzvVSi0N7KGp1nQlKkaumZOUeQ3u+Vo+MEdISFprLgYvz1mOto
+45mAK2ev6cr+uWh/oiC2U67Q+h11ytebF6yqNhvarKfKGWWFhgIWiXRKd6ULuRud
+sQCUBKsL6qSpSvnG6kfgDNZd+NtE7vIrv5vV1Q8C7CHumkVPjczng4aGYYgRrEGD
+pJK0g4XCwXwEGAEIACYWIQQwV+6aRI82LXQgWneasSDaCnb23gUCYiYlhAIbDAUJ
+A8JnAAAKCRCasSDaCnb23tOvD/9w31QRo3VZofiMUMohJP7F0udIu+TQjO2EeonP
+Nb2xFBgg2+rPz0SRNSuN6VtbchvjVhU02TdLuE6bk4mb99Ed1C0i3O5j/0RFp8Xn
+49SIOPlNENrxtFXuwIBAujO1+vWjNp1sYTt7weutm/gNxnXSOgKktFmbTiNAwF5l
+xpoRrZaHtd/1VbhoEn8NTkN6+wp5NGkK7C3DaUR/40yXc+iA9r2CWbuWJC4MpThm
+V//fVkK50BvzQ+6A7NYzaLUMbbob9vvP7w3n2bp0FhZ5K9ypTtn1DitTttVH/qdJ
+PwbIshT8+wNqW8qbYHc0Nxd6s3yVP3NTVAHNPc7IvpTL9TNeXpienEVPyb8abi6I
+tL6BYPZNeHkEoay5TjMgdqSOCUDsa1vo8nnr4+TP3lqnHu8CGacnGDDuoBcoAfRG
+eDsjER4vwWPru3NZNA+zQ+4C3LuoiBt5+3yG+G2T298VlvyTduFfn1v/5M61tM4K
+NWGar1tl1rDrRIGXcu7JSWeUaauq8/NVR1wBzbO+fGMS33JWKhL8zzM1EL9KZZtq
+Bn/HJ/aKIQBujTLXa3eF7O10byMygysiaZQS5t8+IB8I3bKznM+JdbaOKjm++imK
+7bfLft7DfOgBmud+YVbQPwDyIPaDSSX9ixvZ1chihxDjbcAM3USb/7V/yPnc/oOB
+8jThE8bBTQRYA5J2ARAAxxEMCwZX6m1Qs6u6mHARRU3NVggBbaoinoTq7CE5yrrG
+cOJ4WMJSkMXLpoI9NcSToW3xwQgXhpQ8nWKnpioOCH2Np8Esj8GWWDCZ+y+wBqwn
+sra+FTLwye6O6g2Y44puIn7/hlMHZN6ZV8eqDBeJ3LXrCPGgDri1BwJ9D36xnzYV
+qGoJTioHTiOgxo4SP2hpL8vo6yP/l7HTFft39UbhDnyPmLuElxYycoQJqhDgUccX
+R6iq3iCKc9y4N/MlsIlRn8IEE/pxSM/QaRx95VoNbkvMTN3s3HgZkWFfM0/T723m
+eewZWk0eqy2nuUWMFd/ez0iwaqi5AeR9Ew0VBQZ8AbFKGy3m2B5iQjsnb8jb+QBZ
+C5i2ezuI9oxaNVj0jLd04TOsEcKq5weBovuqJ8q2XxNQ51WTKm8Wd1V2vhOEuWMK
+3wi1BKxhJ/neNDraEWZkF4UT1QLHY4pjI/aBTPUaMUoFKw4OiGt59CKyhW/vsxwB
+W6pEeYOKs14jjBButtRmBHsdrAxCs/zw+AlSpTSPXpXeE6B+fdpAMMyZnfrAIvkX
+6LNDRM8bhXdHpADnlqFEaNM6GE0lj5TDphTL3KR+iz95XiZmO8eM/29Z4P/u1Jyq
+J1QC1TxW0WxJTpCHwLSynFBKH8FdY6+w8mBj2pSbYgyT6JgdONWRwo9kccl2P6EA
+EQEAAc0pVmxhZGltw61yIMSMdW7DoXQgKHBlcnNvbmFsKSA8dkBjdW5hdC5jej7C
+wZQEEwEIAD4WIQS2AGRgtgqA54IGJEnnR98flXWjqgUCYWawGAIbIwUJFNHfoQUL
+CQgHAgYVCgkICwIEFgIDAQIeAQIXgAAKCRDnR98flXWjqvkYEADGQ1oHzOJNVYFn
+mB6pxB6ioJ5OtmMNnWljfYFuN58ZvTPR+ZHMf0RJAEYf5vjo87tyJs63exh3hlfw
+bEb++fCxTd06ylohB8cf33Jev6wJGRyvaTsNZMwVJf/kmSGWui4BRGkhE3zLj447
+9ORYVjccAgTLdWSHAiglJQQN9Ig1vUoeei03MdzuzVnfJvXgrXhdX8udAv/81PPf
+Y0EZIMC3cLf7DiHlYqGD6Q9y6ejVVVQczgByqouSERs039DBBQtZutAypwH4klaU
+SBKqSGfaevD+Lyr62UrsUGLgOEgP33FGWO3Ydw+3uCqGdn/Y7F+gcYe6L98U+CZ4
+r5z/AdnbYUfiXLRfCxJ0iOySFF/9c9NoGWB6it/IgQEUF4WSCN0VQ1KmXUxOyjpl
+jFeQuDVTEQW0j5fkSHvOmKt+zqPBzjhowcppsljdX8wGZipQFFHcfrGcgV2B+NV4
+UBHrnDswSUldHHaS8nmLGlXlbb5T/8y+GSk3pWTVjYb01eZB4ZfPaX7YD1XiqlOf
++dUAhyFgW1U/OwAmXI8CoHsjjm40xA5ywZ2fhbmUuAPgw12DAhRx72NDU6gXQGdl
+IIkYuNTRnFVQ53t/AX4/5jxEHEEiFVy/FqW4jdiZPSuioJdolpgYBaBdkIX2DQ2d
++9HR1FFlFR5P+ovEk1mSwr6IW0siW8LBlAQTAQgAPhYhBLYAZGC2CoDnggYkSedH
+3x+VdaOqBQJccncqAhsjBQkJZgGABQsJCAcCBhUKCQgLAgQWAgMBAh4BAheAAAoJ
+EOdH3x+VdaOqXPEP/RBD60xMLKHsMfD2PhzcbsT+4hmvO9gJQEGgmWIOqHp2D1x+
+cAk+Ck8VJb3uuMa8qzi7RTKGxWQWrmat/C1mq1vPDLSIgjTEe+eTD64BsXJgo955
+5dMgQLUvD8JorsrAJHYQj65/JWKAQr3feZsYB96XtKLkl892vqvgIt6DSzMazyho
+H8rQRqUbWvd5O3ibB0AeIYIHgRsbwhM418eOFj8xAbx5EROdNC0x+EV7q4HJe0gm
+fJ34a3SWUxNtYfXPFF/H1N17k+oiQ68FbLZMnLTJ1q5MKCSrIRS0WmyzqzGZH2jL
+viS9FwzxauBeXOg353vjRpc5zWFWjGzczdY88L8k7jxwKYLWTDhTU2mmMhAHMyFf
+4XeeHjJ/kfFNDSXWW2OOAhEHM28nmBpIvHddYoMBjXGc4KmSf1aJlUFfTwrdOMA3
+WbooPKSXLalAULK7owkNmyZDrngTSIY0yPMRYPIArIOOmMpCO/ilStsfThQhj4uA
+I3SumQvT6T37MvmY4keD7eGwdOrQqmhbyo5J9hnPrQD+GR9eSXf3CPMV+4217BEv
+M+arbaDwPW5sYjWox+ED88wsq4rOK5CR0+pu6JS2bPQW+VCSy4VyOVIlj5nAnzpI
+ec9R4pE0Z6NJTZX9NmcpRZFn2q+bMAiIoMPPLSKCMLvMQh+G6fUcftpN8RAwzTBW
+bGFkaW3DrXIgxIx1bsOhdCAod29yaykgPHZsYWRpbWlyLmN1bmF0QG5pYy5jej7C
+wZQEEwEIAD4CGyMFCRTR36EFCwkIBwIGFQgJCgsCBBYCAwECHgECF4AWIQS2AGRg
+tgqA54IGJEnnR98flXWjqgUCYWawGAAKCRDnR98flXWjqlYyD/9Teb9rKFn+09jn
+K6JhZi++kA6gCMM2LbsKKJBSvFAYhhAMPpc/HR0j6okBfCImlca6Fniqe7xyo0nk
+Qq9mvXirGM03c940rcD5ymRqT+TjCW9seXWTccimi9RconatDfSwyJMWpzDkT2FQ
+NnNbR5hSRSaI/BJlISGDfIgJeWRtrt7xCyPQd2bzSW/DiVMQDnLz1crsSjM6SWNg
+vCuBHNo6C7oBvmgqGYew04uBDragDsa/43wjxHTxxCChDnal8uQ7aIgOXurhwPg6
+bHgQv4zNo67zviEapxb6+zO/kMn79GW5BXn7CaebY6Iwr+sST7c/wSsIfcMTbVCa
+2XxS+9Trc+P9GAMIl+M/pJY6KcdIuAC1VDolm+4v9j4j9t5+CZzwHy3ZQpZPQnk5
+AjrtAgIAb6yotcP+3hJxqjoE7uDmhnxWjrU47haiktMDnQURkk+Yd6So8aw8TfB4
+CPmaRdRHzHeqL3y1qTFSMBPXFbnopnKQKn0rm0BAk48J7A0ps3NumqWghy/BWl39
+qRvSebg3M8XXFM1Vn4G/telnYGxaKifNjtkgsrE3BJKzR/JtkeYu4gmjuVpUBS39
+hRzgVi3d0Y1dc7QiJr+HYubJMEC8IFtvsItEKMFT4D3N/CQD2u9yDUYWjytyVHMK
+jRK/SVDX0Rsi/nk8E1+G+WCh3XSfSsLBlAQTAQgAPgIbIwUJCWYBgAULCQgHAgYV
+CAkKCwIEFgIDAQIeAQIXgBYhBLYAZGC2CoDnggYkSedH3x+VdaOqBQJaDfDeAAoJ
+EOdH3x+VdaOqYPoP/3pThwNkGsdDkuGrJuhxmv83YhLlavZIssHiTYXn9b/BsvTm
+Crodr3/ZMypejFGSG6D4QAcUfHKU9HjfI32vPSmMFceutrzsUA8cyqr+qqs4RR2G
+TK0SprpYiGhqgSR2FoyFTJy2NWMvZZnT4xZ5GOp50tBf5UKs5U9xVsut5FGED03c
+o+A3Bun3MKDqEQ+LiXbNkp50eRM2MogDAnTFmgzAVrVj68vwSbw6v4gFm4YEw/9k
+y1qw84hBpP8a+zLHUMOivCygJ1frNpNnLkM5Y/cUiszO6z9gNvPABjFKlaT+MXjj
+JJucnN2NTrsR8bdSuwr5FEbDxomqzf/iocpkxlmc0v7jAmkR2dnIWJz8xdzDoUGz
+NATCsg5ozVYR/DOfYTFRiTakznNoHCrvo6CuYYzMwSJx8wPKouh3ARCpHvQ+9FuS
+uH5HTYumHBD1zkAVdysuMoyZ5CVtBVqtNSysQtxMrqsSULWiNLszfuPRfMUT6SA1
+afZ/IFGC2cYNYRruv6gNq55cmcR/T/hbDbwjDFqQ1zuvQ3jUYgF6TBLEp2R6koVm
+C1UYZIOUV6LmFvkstYWWMDU5LsJFUkqlsdp9CN091vW82pkCPjLZIle9VWkjUo7m
+Sr50V1iKk5G/Q3YiGjow1htBRPUQF5Tjobk3S+IKgtLXn8/EuEgCj4/Mqlb3wsGX
+BBMBCABBAhsjBQkJZgGABQsJCAcCBhUICQoLAgQWAgMBAh4BAheAFiEEtgBkYLYK
+gOeCBiRJ50ffH5V1o6oFAlnIz+wCGQEACgkQ50ffH5V1o6pvJRAAkxE4HWvOUgKZ
+2hgeXQc6QhX+HBi5QfPLgC5bC1BkCTcbXuPdKuZZPdg5J/+0GGbtqWSJnXWXL1D1
+VZWs6lpgM9Aclsbn0XnsHSX4ukyafrR9v17CnUMMXgXoggTs1a3jd4xva0QDQq9w
+Dru94cZU6rsBsjimeNvfRkiG6buAHqZJPblCQ6VDI9OPopHpk+rV93Bj3gLVnrbJ
+AKNNHljcxFRV7D1XC19aFJjrGjEIYBCvxK26Mg3VYMm/jo71nayFWyS6/ZaEPTCV
+QCqrgX6+4ACXo/tMTSyFQ8yUMXKmqMZXNKynjl7KeRL08iC2sZ4B5e7ceuu95xv/
+4DzImQf5ETBbASxiyrSHRRb60tDskwjh7pFxZGHW0G/HAT3Ss3wPbksPwxsMTIfg
+OLBqSiHw2ZS92Y3DQbU1KH1XWSYdhwCAm9qF3kKZyZYOiKDI9KZ3wbF0e7p29cvk
+mJ8DbXtxmLps7J7w5Oe5UIqVHr4GYovtSuzRS+AxUEpBWeQOnxKNe6/oEHmuLGdd
+HcE6U8wUGGuz86gnhU+r9tfSkA/aFUcdBIiIiqYSzMpqjqNGKD727k77DbJzbuOj
+CXF+cYiyCK/KvyYc3XMNK71XI7y9valWbt4zNyjgkyKjiZScCaHv0A7LDalF7gOh
+spcevd0dapstkMv/HvJg890qO7Ox6NzCwZQEEwEIAD4WIQS2AGRgtgqA54IGJEnn
+R98flXWjqgUCWcP6KwIbIwUJCWYBgAULCQgHAgYVCAkKCwIEFgIDAQIeAQIXgAAK
+CRDnR98flXWjqil6D/0dSV7kphVFpx1mrqdWGn16yTRK+xMEz8LuohbMnSmqu977
+r8EtUj3p3z/MSlP1g3v/Om1GCfNl1I74Ih8Qb9cyZOD5wkavb6rXUqbVj/gkYahk
+GWHU9uoP2b2diIKMMi3kVmSOSh3z9yrM9RXmeWqECAeoe8knNydug4CZ+X01qZ9Q
+JrC6Vf0c4Qo/r8n4GDCdG+4cLAFqIvfOn7mZbLjOQGJJ+8PlhHsBPCF5VXfGM+et
+rq2myHAFismsjieGhhBOrd6JoI/+v7wDEkjElxVh0HrBrT9WDdM8ZuzrNS6HxmgK
+H37C+CR+nyInX4I8Urghl2Sqz1P4tX0Rg2HO4sDN/5R5SFPtm1sycE0dRHtS0OGD
+paCoN87ylgcLw8KHxpFZbWBA8FeyW/nYWOnLH+BU5iq0wCksXTeutG7seGciGuIC
+1on/IlemIjAqc4/niIb0tqYQQ9PX/dkInTN4mu29v4eX/0EDceWiPC5cHLtTBjKa
+kU7Uo9mUdRkrkUJCLBN6vdD9rr3uvQZ1jHxlpaAVN+oMAElGQBXB+8Zvwpaf4/Uv
+rZqXWvMSgV6DglfFoFF8pGvDUBKHjJhukvz9GHSPN62FURucrfrOAKkuumppqQXc
++EXr2YOrF4PItpX9VEk6Tz7T8Z2d9nagxNMRMf3934Yp7O9i+1h+MDaa719AAcLB
+cwQQAQgAHRYhBEqLpIwq7ZM71JXFCaH7pffvjEhpBQJahrzjAAoJEKH7pffvjEhp
+i64P/3wbntb/4B7vFUMoPXblRdpv10gHc81jwdih5PIs7rhcLWOqTALgCGVecaHB
+KgwlMk+Mt+sbe+G6lLo2ypRJbcqYTDKwmNQgJjfiYgO6OzW4OZPIAqDgDK5hG2cl
+992V5g6swJvDo+QDUD2JqW4Po+AxdTbBHIZTEj7oQMacFA2BwNgWH6DxmaWkDuRd
+46kWUgrOERuMEvUlDEh9t5U0qNqQsjaa9p7tv3L59BNplmTYrgArNpVUqObaQyYr
+iSPI90szwl0cMq4TGgWplklbS36T/iY5eTASWpapNSQf2uyLHKeghDDE1noBD/1P
+5QiU6NnZzfo8BCWZoAHR/64rpbg2UJbeS9aYtOR1crhjTQVdq6Ba6OB41G4w+GOZ
+LX68rUOUPHO4AHxvaIi+Emo9cdbcg8lYn6a44TvFvhRddUPNy65fL8f8n+SQ6THP
+2nK/qmBnL/xNSBZ4MeC22ChF4kdVaFr2rxREw7TB90UhFHe0u+r1gnl4pBtaVhGD
+L5J1ctEmMpKHW7ViBHQhi6zpiaiTT93jt5SSoL8YRogrB5sQwzCH7vTl1VGp4rT8
+j/rA5dxlt2ODaU8kkfa8RggnLFIF7/aAHPYZXGVGuGOZn4GVnwYkB1i/rFI4uuVc
+adCzCEPu5QjfeoHJK+JuWXaGbpbqXP//S5NSYw5f5GlLzLXHwnUEExYKAB0WIQSc
+DCLV4+zCjGNYJITz3gQA4UFO6QUCWgmkJAAKCRDz3gQA4UFO6bFRAQCIdSAKvVV5
+iqf6fKmY0ZgHSUk7z4SFqfkQsfmvt2A77QD/TYtaRw5ru08aKIz963rkDndvcef7
+PyY+OOdR3J4fggHCwXMEEwEKAB0WIQQwuTPYD849mBotOPsMmbcO9Py7BwUCWgmk
+HgAKCRAMmbcO9Py7B/65EADBiAbUo6y28QUTZxbpG3PhqREHX2rUhb862QWRYIXZ
+8Bk60JAijY7yXEbA0r515f5YDZuiFWBxTj4znENZDitjEd0EUd4zvsPBnG7i2c5N
+qovK7VWJ9hm8fc4eG1kQb4+zGbJ65OnbiCWTphEacXcCZxpc1K3Ao8N7u6KyEV1x
+9UPMCMWYa+/yVtTR102hwn6XVfcZSnqteUA/xJqdLhHi+glDX7ODzzL0XA21Y7eN
+4rKgzdHM7AAETYRpZvncb8TrDZocZhk8dz0Qq6IKWIRMs9ON6Y3ePMo0i9nHB64v
+o/YfjsoGDhI+Yb9HqR7B2ScKyPydrsqXg14AqNcM+ZZG8TrE8hZ4MoglNW7gPEqm
+dzDPnwOHGimr6ADe3OCD8O0r/5CBqHarcXk06Z3ZEUuAF18yYsaUiZK5blUUCTRs
+Xlrf3yEp37PJ6h48afapjFvYDRYPt7VqmQVE5ZE+7Vg++LmbcDOQxaBUCcUK966r
+1spelV7DsF1zEX0sggXMr+qvR3IuuRfaklFfLXUPlA58N7SsHLgPbTDI/vnZK6Bz
+k5gyJWtsXFAaUxb+K+fWX4ki1HmSOmZeUdmPnNkiysmX8QqhRcWqcT94a9ymns1e
+qn61JsEyI8p1ctlneKqZmRkrpVGhw0WW486yY9jSJobTCPJQWqc5YcRZmMXuuiTZ
+b8LBXAQSAQgABgUCWcP8JAAKCRDOjdahpQoh5K49D/48o42SGekUHQQOIyD/RXKp
+WmPmzUxheAwQ4KFP2jKrJISHNFWYaYV40+MQav0ZOZ6EnzyMpignEk3mayytXprq
+mx7ZE1aMg/ZmLrMycqgDYFLZaWJ+HOhymQp2gPmNAryjjoqDEhbFbVbyLdrSkd2E
+vySJX6l0OF/xNiexk+Q02jGwl1qDDzoEmM5zdDoS9L2QH3PRR7ZZdeBbHV35QqSS
+iaM6cpqc3MXDYaPRb1r+kI9By8PG+LYCWzXLBmYbpx9XBuD+Vicp0zImxVOFLJsB
+oqt87D1O8aQdgn5m1ZN8988U5yY+3EOc3SljqPZ6zldQ6cEPFgdKuEkHXwWUAv/c
+JSAn5dhSnT+MTVo90NuXo4IUxBijwyQ5AkCw+zYEq/Rlcys6YPq3vyw+JvlfzkBo
+s9NzHMIpabtfNo+5ZXi6Zir/Q+H9qZCW1NBVt48Adh/FK/XrRku2N8/ezJa4WyCi
+tg0wQkeVTvOHM8qTW7+Baq6b31n8jQcWEnHKArVgAuJejM9HR3EhPGYYAwFwLh+i
+dljfn6e09amjwaxyNpadIdXo9Mln732Z0fWjvHJ6tqxT38CpA2bT8zmYNuCqpVbb
++RsOWi5+cmEyWnp81pVq/xXOs6tLubQKK/5eBpgb9yPNcYmC4U2/cZMxjrWOYoEh
+eUG3LI6ZOI3Uhn6vvVKRHc0kVmxhZGltw61yIMSMdW7DoXQgPHZjdW5hdEBnbWFp
+bC5jb20+wsGXBBMBCABBAhsjBQkU0d+hBQsJCAcCBhUICQoLAgQWAgMBAh4BAheA
+FiEEtgBkYLYKgOeCBiRJ50ffH5V1o6oFAmFmsBgCGQEACgkQ50ffH5V1o6o6TBAA
+gpW1Bid3tKdTbctC35EbKFmTKMnlkqAQNKMFLEDsqriimko/uNKhzAzoWzXyaC6c
+BbFOcyS+CNq4Q5Vvs2OR3recdWn0tJRMc1yt0TD1sIVMDJlnr1KVvlRY7z8gYXZR
+D/BNAMo+s/6eRhDfWfcq7mX8/a7uPCg78FGM/39ldHAD6jiHEjOnrhi15yuBMTC/
+vFoqoJobfKgaMw9C6KHcgFjEdQ/l5ADEfroMFH7OOPfse88835EYfAn4IAfZoDCt
+Itm+ujHVhR2FtnPY3N0iBGlmTOeqRBvQQmaXKIohoAjjFkOwhJ9nTnx9gIdWYiAs
+g5y/V7xEmCQaES4jRnC6w99Ric1Kwz0S8yvX/I+AJVje2zRAsOaZj2BNr4sxTrmr
+KIwVeoZYPnNG16UEism4NXkG9KxOcjlqwNKT+cjXOmSAcbY22C28JlpbW/ZljM51
+UjX3I7wO0F3t7K2GCwDltSj89i8vKgk2V3KV8dmdppOBUzQyI+JmkMS0yFER1h3J
+B618RDcTFzXG2meHeC1c+X9UQYbBX5gF6VbvxU71yD5SFmjVZUA1gTMZVYnaXs7Q
+eF4D1KilprTDcgk7DT3rsa0YsiJtnHFxPn3Ljw7LSyHkHV/lP64o8dX+fPIup7dq
+A9L8vbRHIRXonM6pdWM2DTOVPLv6wmIjsZkFn+4GXnHCwZcEEwEIAEECGyMFCQlm
+AYAFCwkIBwIGFQgJCgsCBBYCAwECHgECF4AWIQS2AGRgtgqA54IGJEnnR98flXWj
+qgUCWg3w3gIZAQAKCRDnR98flXWjqs7GEACZlVtvy0Q45DrRQJ2B5SAeb0ZJ5OZQ
+FPFnnl4UjL2Q9A1jglzjftbhjfwf41K9ouUoa6R8X8nlpGwo8DSZwXNYni8AXUMY
+h01VgSFop/6Uxeaczyz+X6/YJ5Q+UMEkVz2rrezpZXG8pj0+yf8fGbImEqGDJInQ
+ZoJhUDaaFSiyFIMJWQUE52O117fAUvDDfVdvg3PDjaR+Mqf9w6bZNm6Sr2LCJrxT
+Lr71PcpZC0nD0menvUkAzwe4BzVmciSZWtyQB0fhlr6cBGb0WpqgYlXOV0TecMtA
+ZGKrzsT48fspeBGPPobW9t6YsnFgQQB1V3ON4VxHxDeD3OV9Aq91zLl1cgBmp/z6
+5APzzqHXthX/meBCzKLO06w82Np/gIeksFA05HbbykZElslbB2eFz8W3tV4WLWcK
+ucDPl+Pmzlbt8XprWE7Pyn6mFp8beZQWT0VOcSTH/UOfEImplxFLRDTLk0wjMye/
+i06XlPu/1nrditHwmlVjFbdc7NSiO8rXdUgTuOEwdZMyIhCB9SWNxZa+7F3kVKdX
+TBytVaYSfD3qoDBP8bhaeDF8K6054uo5pmBXD2f8WGqbuikNh64i1oncmj475uxR
+KkzByrkY9XN9qRKjWav6/ZemxMRgGmV+HHef8lhyLthDvucIEHELuRK+xWmcD4fn
+5Mhk4DN4LLezwsLBlAQTAQgAPgIbIwUJCWYBgAULCQgHAgYVCAkKCwIEFgIDAQIe
+AQIXgBYhBLYAZGC2CoDnggYkSedH3x+VdaOqBQJZyM/oAAoJEOdH3x+VdaOqTDEP
+/imxxu+aGjfXXkHX/UELGHIyvOGEOEQ83yplaHwot9x1xMUmo35ifJUqkNg5bLy0
+5RKM8T4h57OqN2XCCGgPmCpF+tbWDyYHuF63awMiXY+tjrtF/6tijDAouM1jeAY1
++lUWurhBsiYXkXX7jgV8Ts+OwdJkY7r8TnhltpJnfqDTmS40BAY6aja15cyCTvdq
+pAUnYl3Vc2rlxeSv4tiUhcK7JaK8uLE0cAJkE+KlHwraxtKVLrmza56KRRShxVR9
+7zBbykO9oC4qLHCAg3JpkjPV0We7M3cnrBrM3n/O7uDy5QVFlVXEbT7YLHkPTom3
+gcPr+Yk95jyyDud5DUlwx7/lcgV5yc07Mu696oBClv5FAZXRpDCHNppmA+i5Yc09
+EKBeEKtLcIA6cVdx6DZP/VNSqeWuHDSt+59v9XzdS/5EVSpD1EyAWcg4DGWBWVa0
+FN+FBU8WQApmIBENKuhHpXHa5zUDTvWk7QUWypu8BLtr6CT71xG8enX78v1bVkiN
+gmDetPfEs9eNgREZHO9U3TwQjCPMxFmpBCxTnTKsuYTnXpooiCcScecVE1Xdfmwg
+nlVldVMEb5Lx3NTiFRat3vUE4FbfaYfv/4QQEBbDmOKwk/6sb6yW1+HSDCTejISm
+xEF/cYo+7RsbKpC/+fsNeoL+6bFc59IK9uH3abykibs+wsGXBBMBCABBAhsjBQkJ
+ZgGABQsJCAcCBhUICQoLAgQWAgMBAh4BAheAFiEEtgBkYLYKgOeCBiRJ50ffH5V1
+o6oFAlnD+lACGQEACgkQ50ffH5V1o6qgug//dEKAwRn5Z89OL1vUtulvQVizD2Pp
+X3QleqK78Q58o5NBx/HWwXis0Q/2mjokOGxTtnjzZI1suzrhbXXqY4kEmbqlNopL
+P/lQIwEFs3DMQKAWpvag5E4JuMyPFoX/GzkeBAd5Q65zw1f21C+shQvORHqXhlaO
+k0t6syKTtDXxv06MDa3OtB9rGqmr2LEEwdaRxcir8qX5VggHkZ/2Kr18P3CrV0P2
+QCNwbmbHpYRHk2DGHjlrXbtQ9calHHaLZzkABNilm7qr++hUF3TtP9/UtGJFW+dt
+ACWimKq3SLE/LL1LyOSzaKJdDKSqfe8zlVlvIhmctsYpTaslueJT/BRQaQQlyUmW
+u9cAzWlado83e2Xh2fbvM0C+jJDnHrt0gIRiN+SPa0n3DtWeBOT8i47cKoicfxQ9
+k6++CYG4CmPJO2BBujWq+OCxg7B/RWmyD5AKP24NZgjEdW0f4yqSZBO50YU5QeaM
+cE/2JlOBXQENzTd4TEft4hSPOoQe1cqb/3QzV3m3Jtopdl05ly0OpIiYe1R/4Gqf
+EfEMavYgVzR2cAsQm8qTiZGAo8CvEE3+FHofE8jtm57Ie9N6+8r6NQwKD0qWxnO1
+1h279pqlvZVjGoFzowRfWpljSc4W78sVY5bS5obfvTRBXg2N0iJfIEfv159j42qO
+cOpHb4PsPQFzu+zCwZQEEwEIACcFAlgDknYCGyMFCQlmAYAFCwkIBwIGFQgJCgsC
+BBYCAwECHgECF4AAIQkQ50ffH5V1o6oWIQS2AGRgtgqA54IGJEnnR98flXWjqiyQ
+EACJhRx41mwz3E4J7Lm65nDrCQ/Y8jCUHi6LzthSR2Icyr+DSiXSb5ngsTZFj/qS
+GR1GdpQQ5MbOKpiKw04ziDRB21tWw+0svijdCd/ZyLIUF4EF0Z4q4JCD65E/bmjz
+d80+fmLVOnns2eBdAdzw+RxZBP4qojsHsMLTDg6YrPgpwz9leQkJVJKI/OJGymWz
+Sohi+I7a2wjra55+8+2Hy14AC7y/LjTSeqSlgyPhkE3/WVr4ou6V4lhR9SRPqIrF
+hL+s6p/XgiCQ+W5oo+VRGrTFNIk+UcbdEPBRzS4+eowGyVT44R5dTTOFUeeAZYG1
+Cx6f0B3dxDhmwqFIEA8k6mIBUsWolv4fT/c+K5uh1OYNXvsw+iKYj/v7oraaY4Rp
+WPbOGMRMgXcC8BQUChbBX/Ik0iXTz5o6ViIhsHbU9OT+eOREjtyFEGldVBwrPrMo
+dEMy0r6y/HlMMVLugLAUxjuHQKiK/GQCk80CzSWczmSuGDDHP2dlYN1KYMhLAwfx
+e0scES1sXL7MlGD6KtTiM2A5bKYENmvIzZ6ae8AY4yM9zpdRgWJnr7dtLDuiIdLb
+IiLPVhfTlPBzBjmeERZ3ry/fJ6R1mkG1l3Jfx8Yl7tTFqTZ2UAwNR4GF57pDtNiJ
+O5x6+Oq1QvMoKPm0pJj8wED5mNSpt8ngq3DoWy3Na22AScLBcwQQAQgAHRYhBEqL
+pIwq7ZM71JXFCaH7pffvjEhpBQJahrzjAAoJEKH7pffvjEhpA+0P/2qhQcQbCz3E
+ntrh7oPUe5xgE3byHDgqJCcMa03/ILIm25mNC6cXoQCGB2V7MalDKbnasf7Nm2L0
+5Oc3VBL5kr0QuPa2pzXPo/3PaVwdl+UwtwFGrtL7vkltXQN20kkgxtndHsj/pAbs
+raL0lD5jrR20aOTcLHcproPADEJJp5tTMFUD/DBZZyG6XDXNkbU8sVRB/Sy7O3YS
+icLw0Ug2fi+2uP7yTfHOMcTq23iQWrL1R0t2luKNxWrg5pTPkxqhSoIH6xAsyunY
+r5OVuygHClWtECVGcRoRDCddtNYJNN/OsbhzfSPKlX1MrJ7eVsZyOoBvenkYfIZg
+Adhb4DjgfOfDfQE+LkVdiCAK/Eaop/pU0IoyYY4DNOy865CakCYWV2/NEuGNdIRV
+LiIZ7nNPDLcKIv5Tnr7yyxrfC2MjxusZheO1Wb+dnJPHRNIPOnTR3Vn2O4Zfv+Yd
+hO48X/jjoAFO01WYkzZoIvJ0dlu71Ew4d05tiBDp8tUNOTNmqdNk0SVwAuU9CipB
+mPBxl6Ru87DR7P8ppZzKuxG/f6uLNLawGYfTS2eldqENr1x8Z5lVzc9AJQ8K+liK
+MyYsLWP7v8L/JlSqRgs3ySKqGNGIZPoBHIdPYPYza4V8vd4I5kjHS6+N9AB1sWCc
+VlDmXpYYTyiw4uxNQqqar9Zc42SWSUJawsFcBBIBCAAGBQJZw/wfAAoJEM6N1qGl
+CiHko5gP/0U2Qzh8vmhajvkftFtwzmc8Woi5QS7OJ6zfjPo0YzqMmjSVSvfa4W8o
+HREM5RJ3DA9hWNrm8gj/Cv7KkXECS3FBrkNHwbC2j2SXQFseqWdi+8bbl/g3hSWp
+jgNOGQ5CRt2rILaHZfOtSAja06Dmoe+H7UZvTQueTt5Au5dGb1wSWmqtqthI4oCw
+OJoMPU1bR2snzYlhipOUUlpaaR57mg+stvZFWIl2Nm6knr0zpJeVY6NpAFUWM+kh
+8aX3pNwApANxETqpDATLJYnABw6xgj/hAlID2B6CRJn+DAnahZw+sBGw/zBPCm0D
+hjmMfR8otEvZLLW8ckT8F6sAW7WYBrQpDD3OK4lBAtqmeBF7UvKDPcTLJfCAlcoo
+5LSWvZLv3P/C7w3zzkBIMcSHyAPQjh1MBOybKCKxv61Dk3qo0cadwAMv2B+qi7SI
+6IXqVf8b5VQ3zs5vhhMrVPRKjMNMe7XeMGlgsW3fPartEtBvOQbET1c9bBfVnBRb
+kOIVoZHXYxErxOsYYc07ROmuS1PvMsnEQYLYrkyX5K6ANPvmEqg7juaR1J4AiDFF
+acr9beGrY/3mAUACMHbcKlZ9dz9SyWhcVhekAUr0/6yOGhoR5lHWTaObgmAO3i3z
+5OMM/p/kQgmbMe2Of8MpMWvMFIv+izQ6ntQKbSWL0DM0WluiYBDozsFNBFgDknYB
+EADIfDDeG4sS2x3KqCOIYyG0Qo4uanvIWXjnQdj2g9wdLTZTkELXfvSyw1KGt04K
+LYAINt7jVnHHzlZy5o7U5Wq6eDa6ZfTjuaPbFCyJMAVEMtmxC7xxAWgTL8MEvZuT
+tOF8rUuH43JkCzRRo+kOQah1LQN72iwD7hPo1/g85hp5q9thUVJ0MpLiVRugGoCk
+qcP9PMoXEVym4XHoU33aW15Ns6iMYyIuhm8Qtjzoqf0q4VnwUzZlI0fHVnvkqi65
+8at2R4KhlhViK7NP5+e3aK3h0ikiWcX7LEy3ksH8xKuYQIHkIAUm3HO4xxybrINR
+lBJZaL29SDEgnC+3nDQIgFPfncMZ4/hacfSvligEqhFuHuLh9PSVa8xdMb9XaWrL
+MF0KUx5vId98yBtrboqPqh4PKAN8LloaxVqrDVR7+pDlBGpMlwlXvFUtO1LURqZW
+qR8rA9LTNN+esXWNC2c78/iqR+AFFEb0FcYavKQelc7KqZfpnTBtXY/iP6nTahJ5
+wOXnJVemUl+xd9tL/FedpKRo3/uYJTb26XbpwBU3EOQM+6fO1IWL0r0szZP9dZgl
+AZczvU6mY41AkYXbiXHtan+Shuce1uMYPOMM8Lh5IvswDnqq2tZosz7h48oA1y3a
+rSrpzS5IuzRDbOhyFLrgxyDRjMvWyKdY83WtcsOTeXCkiwARAQABwsF8BBgBCAAP
+BQJhZrAYAhsMBQkU0d+hACEJEOdH3x+VdaOqFiEEtgBkYLYKgOeCBiRJ50ffH5V1
+o6q0GBAAt4KFaAcQZ7/yfApasLwIxDIrKIlGVJ+8RGLkzjQ9jHAoLsgej56Qx2Ut
+zyRWS/lZKrfNBw2hEyMe61mUJWkYlq3k/CIKKd7ojdnkl8qxTt8kTgpc8PSOv2QL
+L8geUJawotlMcfQs6T3y2plwVDAUzmdATO7xSzzoHGVmzkho3SMgsPapQL4Rh6DS
+F3STJr7XpQ1qj03EJ5QShLDGG0HgggwIlNQ/HSlYSsIfF4JSO/E93aHkDXToNP/d
+9ry6E8ev2Cge1+t5XSyga7G7WuoasyftdyiuwZ231dfi3I8dxcNcJkwt52NhO7s5
+xhDreGL5lZRck+9wisHhK0K6ZoUQLAuh2l0NYHufo4Xyv5f/949uNLyq374DbQsY
+PiXsGtbEN7KaueouZY3xEF/bFXjK1V9xHHV6twUwaI1FS8ZKH6SA/onsPjJPjEZO
+r58K1LoEfJYwHSWhdxa/Qib/rOekilRy9fIvryzgmKq6S33isqXBv8gHoxJZqefG
+TxTNZ8aAVd4ZStHxhYu5BKjboBx407K7aAxb6T2RiLSyqQHMw+TXNZk3+BI2kPK7
+sBYam/hG04qEmFq7HzPC3CqjOpLH6mWmdjPmJ/AI0y7OczpPSPiawpCIrU3uUjAo
+EZpPY/xIn95xonK50jPh2zb4ZSXp/GpJHbsSCz4RRjQGZLvARKvCwXwEGAEIAA8F
+AlgDknYCGwwFCQlmAYAAIQkQ50ffH5V1o6oWIQS2AGRgtgqA54IGJEnnR98flXWj
+qn4yEACA0f1XBAg+WMaNPtIt0k15yFPfhdbOg9GhDcYGgvFIOxRuaFWw9SLUt7OG
+uUnIpKxKRXtQJss98fHkijo70ONYWPuLhfRGK/wg9Ao6MuFw5G8m431CBS/awrie
+b6iPjvAARXJCPTTBZk/NC988jiKdCh8PbTCHDsl+gSDytP15QUrdqSfS2Wf4653e
+j7+jtuTjxZzmGgvNSi6JDlb9KNtmBQKQAgpnOQM46ItESmzHDnmdcvhPLUDsjwkp
+IJ6clasOzaObwxJiba7iFPcGwcClCSwYjMNXFtneCGUnEAa5RBIx+i+LV1iqB3VR
+vTC6tMIUueoQ7cdTy6afNkhwQYXm4/pDmNT8UMdnzwnlTpFQ0CegDQRDWc+dIDDB
+HGEEEYBh2vTOE04KrmYUp1bQsNegPfvLwoHib0jEvohPMJ2fJtZAd1SJElgwPbM8
+H7emKBiTsHwF8gL7G2jo7AoGpqYjqXkCRS0tSLTNr+qHh+7Ltrkbu/ZVTTfh4Q/q
+w3VaLYQh4C0tBma/YevQy1O2c3TZXXFz1QF8b9/Hj/3sq2KgT1AcZ51E+xG+cb6c
+Uqgkihmgm39xx24GPlNAdCRuq01+iILol+Wox6OwF6hmqx1EMSmxcmGoUREr0rkM
+nFVsWeAYeVoE4q689qxCPu9iCMJMJnkRe1o9oQYSN7my+S98gA==
+=srTz
 -----END PGP PUBLIC KEY BLOCK-----
-- 
2.56.0

